cmd/main.go

```diff
@@ -37,13 +38,14 @@ import (
 
 var (
 	bqDataset         = flag.String("bq_dataset", "", "DataCommons BigQuery dataset.")
-	btTable           = flag.String("bt_table", "", "DataCommons Bigtable table.")
+	btTableName       = flag.String("bt_table", "", "DataCommons Bigtable table.")
 	btProject         = flag.String("bt_project", "", "GCP project containing the BigTable instance.")
 	btInstance        = flag.String("bt_instance", "", "BigTable instance.")
 	projectID         = flag.String("project_id", "", "The cloud project to run the mixer instance.")
 	port              = flag.Int("port", 12345, "Port on which to run the server.")
 	useALTS           = flag.Bool("use_alts", false, "Whether to use ALTS server authentication")
 	enableBranchCache = flag.Bool("enable_branch_cache", true, "Whether to use branch cache")
+	sparqlOnly        = flag.Bool("sparql_only", true, "The service only serves sparql query")
```

Comment:
Sorry one more question on the flag... should we call it sparqlOnly or bigqueryOnly?

Because, in bigqueryOnly mode, we can enable all APIs, in theory.

Reply:
change it to bigquery_only. this is easy for client to realize it's only talking to bt, but also it can only support /query API for now